app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
```

Comment:
Continuation line over-indented for hanging indent. Consider wrapping the imports in parentheses like in `main.py` and run flake8. Alco check `checklist.md`